src/PKSim.Core/CoreConstants.cs

```diff
@@ -1335,9 +1335,9 @@ public static class ObservedData
 
          public static IReadOnlyList<string> DefaultProperties = new List<string>
          {
-            OSPSuite.Assets.ObservedData.Organ,
-            OSPSuite.Assets.ObservedData.Compartment,
-            OSPSuite.Assets.ObservedData.Molecule,
+            OSPSuite.Assets.ObservedData.ORGAN,
```

Comment:
@abdelr @georgeDaskalakis  in the current develop, the constnat are back to be ALL CAPS AGAIN.  I have no idea what's going on here but I had to update all files